docs_app/content/tutorial/1001-but-rxjs-is-hard.md

```diff
@@ -0,0 +1,108 @@
+# ???. But RxJS Is Hard
+
+We understand. Believe me. But maybe you're not seeing the real value of what RxJS offers, which is the [`Observable`](API) type itself. There are many [benefits to using observables](1000-why-observables.md), and you don't have to use [operators](GL) at all to get those benefits.
+
+## Solution 1: Just Use Observables Imperatively
+
+There are "RxJS purists" in the world, I suppose. Those folks would tell you that you "should learn the operators". Maybe they're right, it's always beneficial to learn new things. But you don't have to know operators to enjoy the [benefits of observable](1000-why-observables.md).
+
+Consider a simple component that will send an HTTP request when you click a button and display the results in a list. You *could* use [`switchMap`](API) and [`map`](API) and [`takeUntil`](API) and a bunch of other operators to get this done... *OR* you could just subscribe to the observable and manage the subscription yourself.
+
+In the simplest case it could look like this:
+
+**React (pseudo-code)**
+
+```ts
+export function SomeComponent() {
+    const subscriptionRef = useRef(null);
+
+    const clickHandler = () => {
+        subscriptionRef.current?.unsubscribe();
+        subscriptionRef.current = ajax.getJSON(url).subscribe(data => {
+            setData(data);
+        })
+    }
+
+    useEffect(() => {
+        return () => {
+            subscriptionRef.current?.unsubscribe();
+        };
+    }, [])
+
+    return <>
+        <button onClick={clickHandler}>Load</button>
+        {data.map((item, i) => <li key={i}>{item}</li>)}
+    </>;
+}
+```
+
+**Angular (pseudo-code)**
+
+NOTE: In angular you may want to use the [async pipe](LINK_HERE).
+
+```ts
+@Component({
+    template: `
+        <button (click)="onClick()">Load</button>
+        <ul><li *ngFor="let item of data">{{item}}<li></ul>
+    `
+})
+const SomeComponent {
```

Comment:
And even though it's pseudo code, consider going with `export class` to more closely match the React example.